src/BarChart/RenderStackBars.tsx

```diff
@@ -136,22 +136,25 @@ const RenderStackBars = (props: Props) => {
   }
   const disablePress = props.disablePress || false;
   const renderLabel = (label: String, labelTextStyle: any) => {
+    const lowestBarPosition = getLowestPosition();
```

Comment:
Plz define **_getLowestPosition()_** before using it. I mean move the function definition above this statement.

Reply:
I will. 
It would be super helpful if you could provide a runnable version of this project for contributers ,including prettier / eslint / dependencies etc.
Currently, if you checkout the repo, _npm install_ fails and you cannot run the examples out of the box (e. g. `yarn ios`) + the code formatters don't work properly.

Reply:
Thanks @christophby for your valuable suggestion.
AFAIK, the only change required to make the project runnable is to move `react` and `react-native` from _peerDependencies_ to _dependencies_ in `package.json`.

If you had to make some other changes to run the project, do let me know.